apps/meeting/src/utils/image.ts

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export async function createColorBlob(): Promise<Blob> {
+  const canvas = document.createElement('canvas');
+  canvas.width = 500;
+  canvas.height = 500;
+  const ctx = canvas.getContext('2d');
+
+  if (ctx !== null) {
+    const grd = ctx.createLinearGradient(0, 0, 250, 0);
+    grd.addColorStop(0, '#000428');
+    grd.addColorStop(1, '#004e92');
+    ctx.fillStyle = grd;
+    ctx.fillRect(0, 0, 500, 500);
+
+    return new Promise<Blob>((resolve, reject) => {
+      canvas.toBlob((blob) => {
+        if (blob !== null) {
+          console.log('loaded canvas', canvas, blob);
+          resolve(blob);
+        } else {
+          reject(new Error('Failed to create blob from canvas'));
+        }
+      });
+    });
+  } else {
+    throw new Error('Failed to get canvas context');
+  }
+}
+
+export async function createImageBlob(image: string): Promise<Blob> {
+  /**
+   * Convert a Base64 encoded image or a image url to Blob
+   * 
+   * @param image a Base64 encoded image, or a image url
+   */
+  try {
+    const response = await fetch(image);
+    if (!response.ok) {
+        throw new Error(`Failed to fetch image: ${response.statusText}`);
+    }
+    const blob = await response.blob();
+    return blob;
+  } catch (e) {
+    console.error(`Cannot create image blob with ${image}:`, e);
+    throw e;
+  }
+}
+
+export function BackgroundImageEncoding() {
```

Comment:
Could we use webpack server to load jpg file as assets?
Something like this: https://v4.webpack.js.org/loaders/file-loader/
We might be using webpack v5 now.